### DISCUSSION TITLE
How to Catalog CT Scans in EMu

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#336] [HOW TO...]
**[How to Catalog CT scans in EMu - Google-doc workflow](https://docs.google.com/document/d/15hgHwASqC-gSn9ijVF3sztK1RoJwpiXMr4TSG--rHTw)**

**Comment by magpiedin** [comment]
(in case of any broken references: https://github.com/fieldmuseum/EMu-Documentation/discussions/338 matched this topic)
